openfe/protocols/openmm_utils/omm_settings.py

```diff
@@ -371,3 +370,58 @@ def must_be_positive(cls, v):
                       "intervals must be positive")
             raise ValueError(errmsg)
         return v
+
+
+class RepeatSettings(SettingsBaseModel):
```

Comment:
Move Protocol sepecific settings under the protocol area itself (at least for now until we refactor the settings).